app/components/avo/views/resource_index_component.rb

```diff
@@ -135,8 +135,10 @@ def show_search_input
 
   def authorized_to_search?
     # Hide the search if the authorization prevents it
-    return true unless resource.authorization.respond_to?(:has_action_method?)
-    return true unless resource.authorization.has_action_method?("search")
+    # return false unless resource.authorization.respond_to?(:avo_search?)
+    return false unless resource.authorization.respond_to?(:has_action_method?)
+    return false unless resource.authorization.has_action_method?("search")
+    return false unless resource.authorization.has_action_method?("avo_search?")
```

Comment:
```suggestion
    return true unless resource.authorization.respond_to?(:has_action_method?)
    return false unless resource.authorization.has_action_method?("search")
```

Let's just return false if the authorization is setted but `search` method is not defined.

Reply:
Ok, thanks.